sdk/loadtestservice/Azure.Developer.Playwright/src/PlaywrightServiceBrowserClientOptions.cs

```diff
@@ -60,7 +60,14 @@ public string RunId
             get => _runId ?? _environment.GetEnvironmentVariable(Constants.s_playwright_service_run_id_environment_variable) ?? _clientUtility.GetDefaultRunId();
             set
             {
-                _runId = value;
+                if (value != null && value.Length > 200)
```

Comment:
lets discuss if we wants to restrict it more to only alpha numeric without special character